src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/BeneficialOwnerIndividualDto.java

```diff
@@ -72,6 +76,15 @@ public class BeneficialOwnerIndividualDto {
     @JsonProperty(NON_LEGAL_FIRM_MEMBERS_NATURE_OF_CONTROL_TYPES_FIELD)
     private List<NatureOfControlType> nonLegalFirmMembersNatureOfControlTypes;
 
+    @JsonProperty(TRUST_NATURE_OF_CONTROL_TYPES_FIELD)
+    private List<NatureOfControlType> trustNatureOfNatureOfControlTypes;
```

Comment:
got the double NatureOf in this class as well

Reply:
As above.